Sources/Quick/QuickSpec.swift

```diff
@@ -72,12 +84,13 @@ open class QuickSpec: QuickSpecBase {
     }
 
     private static func addInstanceMethod(for example: Example, classSelectorNames selectorNames : inout Set<String>) -> Selector {
-        let block: @convention(block) (QuickSpec) -> Void = { _ in
+        let block: @convention(block) (QuickSpec) -> Void = { spec in
+            spec.example = example
             example.run()
         }
         let implementation = imp_implementationWithBlock(block as Any)
 
-        let originalName = example.name
+        let originalName = example.name.c99ExtendedIdentifier
```

Comment:
There had been incompatibility with Xcode builds so fixed it.

https://github.com/Quick/Quick/blob/7e6b060b41e0110a5be1f4af324000acbfc21ae7/Sources/QuickObjectiveC/QuickSpec.m#L113